core/client/fs/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -500,6 +501,24 @@ FileInStream openFile(AlluxioURI path)
   FileInStream openFile(AlluxioURI path, OpenFilePOptions options)
       throws FileDoesNotExistException, IOException, AlluxioException;
 
+  /**
+   * Convenience method for {@link #persist(AlluxioURI, ScheduleAsyncPersistencePOptions)} which
+   * uses the default {@link ScheduleAsyncPersistencePOptions}.
+   *
+   * @param uri the uri of the file to persist
+   */
+  void persist(AlluxioURI uri) throws IOException;
+
+  /**
+   * Schedules the given path to be asynchronously persisted to the under file system. If a
+   * synchronous persist operation is desired please see
```

Comment:
nit: First section of javadoc should be a single sentence, then additional content goes after an empty line.
nit: Prefer active voice over passive voice, e.g. "To persist synchronously ...", "If you would like to persist the file synchronously ...", etc.